File: Python/Saldo/Account.py
from suds.client import Client
import base64
import ssl
# import urllib
import urllib2

# urllib.request.urlopen(context=ssl._create_unverified_context())

urlAccount = 'https://pruebascfdi.smartweb.com.mx/Account/wsAccount.svc?wsdl'
urlAutentica = 'http://pruebascfdi.smartweb.com.mx/Autenticacion/wsAutenticacion.asmx?WSDL'
# urllib2.urlopen(urlAccount, context = ssl._create_unverified_context())

autentica = Client(urlAutentica, faults = False)
 
try:
    token = autentica.service.AutenticarBasico('demo', '123456789')[1]
    account = Client(urlAccount, faults = False, headers = {'token' : token})
    saldoCliente = account.service.GetSaldoClientById(None)[1]
    print(saldoCliente.SaldoTimbres)
    print(saldoCliente.TimbresUtilizados)
except Exception as e:
    print(e)

File: PHP/Saldo/Account.php
<?php
    $context = stream_context_create(
        array(
            'ssl' => array(
                'verify_peer' => false,
                'verify_peer_name'  => false
            )
        )
    );
     
    $options = array('trace' => 1, 'exceptions' => true);
    $urlAccount = 'https://pruebascfdi.smartweb.com.mx/Account/wsAccount.svc?WSDL';
    $optionsCertificate = array(
        'cache_wsdl' => WSDL_CACHE_NONE,
        'trace' => 1,
        'exceptions' => true,
        'stream_context' => $context
    );
    $urlAutentica = 'http://pruebascfdi.smartweb.com.mx/Autenticacion/wsAutenticacion.asmx?WSDL';
     
    try
    {
        $autentica = new SoapClient($urlAutentica, $options);
        $account = new SoapClient($urlAccount, $optionsCertificate);
        $credentials = array('usuario' => 'demo', 'password' => '123456789');
        $token = $autentica->AutenticarBasico($credentials)->AutenticarBasicoResult;
        $header = new SoapHeader('http://soapinterop.org/echoheader/', 'token', $token);
        $account->__setSoapHeaders($header);
        $saldoCliente = $account->GetSaldoClientById(null)->GetSaldoClientByIdResult;
        echo $saldoCliente->SaldoTimbres . "\n";
        echo $saldoCliente->TimbresUtilizados . "\n";
    }
    catch(SoapFault $e)
    {
        echo $e->faultstring;
    }
?>